src/inet/BUILD.gn

```diff
@@ -50,10 +54,13 @@ buildconfig_header("inet_buildconfig") {
         [ "INET_PLATFORM_CONFIG_INCLUDE=${chip_inet_platform_config_include}" ]
   }
 
-  defines += [
-    "INET_TCP_END_POINT_IMPL_CONFIG_FILE=<inet/TCPEndPointImpl${chip_system_config_inet}.h>",
-    "INET_UDP_END_POINT_IMPL_CONFIG_FILE=<inet/UDPEndPointImpl${chip_system_config_inet}.h>",
-  ]
+  defines += [ "INET_TCP_END_POINT_IMPL_CONFIG_FILE=<inet/TCPEndPointImpl${chip_system_config_inet}.h>" ]
+  if (chip_system_config_use_ot_udp) {
+    defines +=
+        [ "INET_UDP_END_POINT_IMPL_CONFIG_FILE=<inet/UDPEndPointImplOT.h>" ]
```

Comment:
Any chance to fully name `UdpEndPointImpl_OpenThread.h` ? That seems more explicit.

Reply:
Done